.github/workflows/integrate-and-deploy.yml

```diff
@@ -71,14 +83,21 @@ jobs:
           username: ${{ secrets.image-repo-username }}
           password: ${{ secrets.image-repo-password }}
       -
-        name: Tag image
-        run: docker tag lf-app ${{ steps.image.outputs.NAMESPACE }}:${{ steps.image.outputs.TAG }}
+        name: Tag images
+        run: |
+          docker tag lf-app ${{ steps.image.outputs.NAMESPACE }}:${{ steps.image.outputs.TAG_APP }}
+          docker tag lf-next-proxy ${{ steps.image.outputs.NAMESPACE }}:${{ steps.image.outputs.TAG_PROXY }}
+          docker tag lf-next-app ${{ steps.image.outputs.NAMESPACE }}:${{ steps.image.outputs.TAG_NEXT_APP }}
+      -
+        run: docker images
       -
-        name: Publish image
-        run: docker push ${{ steps.image.outputs.NAMESPACE }}:${{ steps.image.outputs.TAG }}
+        name: Publish images
+        run: docker push --all-tags ${{ steps.image.outputs.NAMESPACE }}
 
     outputs:
-      IMAGE: ${{ steps.image.outputs.NAMESPACE }}:${{ steps.image.outputs.TAG }}
+      IMAGE_APP: ${{ steps.image.outputs.NAMESPACE }}:${{ steps.image.outputs.TAG_APP }}
+      IMAGE_PROXY: ${{ steps.image.outputs.NAMESPACE }}:${{ steps.image.outputs.TAG_PROXY }}
+      IMAGE_NEXT_APP: ${{ steps.image.outputs.NAMESPACE }}:${{ steps.image.outputs.TAG_NEXT_APP }}
```

Comment:
I wasn't able to delete the `outputs:` line in my suggestion :( 
```suggestion
```

I'm going on a hunch here that these variables are already in the job output, as defined above.  If they are not, then, I suppose this would simply need to be changed to e.g.
`IMAGE_APP: ${{ steps.image.outputs.IMAGE_APP}}`
It would be nice if that wasn't necessary, but I don't actually know

Reply:
yes, see above

Reply:
satisfied that you tried to make this work.